spec/graphql/mutations/stories/create_story_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+module Mutations
+  module Stories
+    RSpec.describe CreateStory, type: :request do
+      describe 'create story' do
+        def query
+          <<~GQL
+            mutation {
+              createStory(input:
+              { userId: #{User.last.id},
+                title: "Thoughts",
+                bodyText: "hello world",
+                word: "test",
+                image: "http://test-url",
+                sound: "echo",
+                totalTimeInSeconds: 120 } ) {
+            	story {
+              	title
+              	bodyText
+                word
+                image
+                sound
+                totalTimeInSeconds
+                }
+              }
+            }
+          GQL
+        end
+
```

Comment:
Oh, this is super cool! Normally when I pass in data like this I'll set it equal to a variable rather than a method. Does it need to be a method for graphQL, or was this just the way you thought to do it? I see nothing wrong with it, just curious if there is added benefit or detractions.

Reply:
Similar to using a variable here but the syntax for graphql gets a little goofy and with the multiline response it made sense to implement it like this to us.